src/web/releases.rs

```diff
@@ -1007,6 +1007,8 @@ mod tests {
     #[test]
     fn serialize_releases() {
         let time = time::get_time();
+        let now = time::at(time);
```

Comment:
I meant more like this:
```suggestion
        let now = time::at(time::get_time());
```
That way there's no shadowing of the `time` module by a `time` variable, which requires you to know that those are different namespaces (I didn't know that until I saw your code compile :P)